helm/alertmanager/values.yaml

```diff
@@ -142,10 +142,6 @@ service:
   ##
   type: ClusterIP
 
-## If true, create & use RBAC resources
-##
-rbacEnable: true
```

Comment:
this was already discussed on https://github.com/coreos/prometheus-operator/pull/829#discussion_r160369963

> Maybe a personal preference, but I like to have ServiceAccounts for all applications I deploy, even those that don't get any roles. I do this so I don't accidentally bind anything to a default ServiceAccount and grant an application access it shouldn't have.

I agreed this approach so I'd like to keep the serviceAccount on alertmanager.

Reply:
looking at the master branch I still cannot see where it is being used??  You would need a service account resource and then bind that account to the alertmanager resource?

I can raise a new PR to this and pull my last commit?